bucket/oraclejdk17.json

```
@@ -0,0 +1,33 @@
+{
+    "description": "Oracle Java Platform, Standard Edition Development Kit (JDK)",
+    "homepage": "https://www.oracle.com/java/technologies/downloads/#java17",
+    "version": "17.0.4",
+    "license": "https://www.oracle.com/downloads/licenses/no-fee-license.html",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.oracle.com/java/17/archive/jdk-17.0.4_windows-x64_bin.zip",
+            "hash": "4f2872ccc7b0d71b4ce1ab1902ab25cbf2a2ba36dc483e033ed49301b093e0bd"           
```

```suggestion
            "hash": "4f2872ccc7b0d71b4ce1ab1902ab25cbf2a2ba36dc483e033ed49301b093e0bd"
```
Remove trailing whitespace